SDW/alc1308.asl

```diff
@@ -0,0 +1,587 @@
+                    }, 
+
+                    Package (0x02)
+                    {
+                        "mipi-sdw-master-count", 
+                        0x04
+                    }, 
+
+                    Package (0x02)
+                    {
+                        "mipi-sdw-source-port-list", 
+                        0x54
+                    }, 
+
+                    Package (0x02)
+                    {
+                        "mipi-sdw-sink-port-list", 
+                        0x0A
+                    }
+                }, 
+
+                ToUUID ("dbb8e3e6-5886-4ba6-8795-1319f52a966b"), 
+                Package (0x0E)
+                {
+                    Package (0x02)
+                    {
+                        "intel-endpoint-descriptor-0", 
+                        "EPD0"
+                    }, 
+
+                    Package (0x02)
+                    {
+                        "mipi-sdw-link-0-subproperties", 
+                        "LNK0"
+                    }, 
+
+                    Package (0x02)
+                    {
+                        "mipi-sdw-link-1-subproperties", 
+                        "LNK1"
+                    }, 
+
+                    Package (0x02)
+                    {
+                        "mipi-sdw-link-2-subproperties", 
+                        "LNK2"
+                    }, 
+
+                    Package (0x02)
+                    {
+                        "mipi-sdw-link-3-subproperties", 
+                        "LNK3"
+                    }, 
+
+                    Package (0x02)
+                    {
+                        "mipi-sdw-dp-0-subproperties", 
+                        "DP0"
+                    }, 
+
+                    Package (0x02)
+                    {
+                        "mipi-sdw-dp-1-sink-subproperties", 
+                        "DPN"
+                    }, 
+
+                    Package (0x02)
+                    {
+                        "mipi-sdw-dp-2-source-subproperties", 
+                        "DPN"
+                    }, 
+
+                    Package (0x02)
+                    {
+                        "mipi-sdw-dp-3-sink-subproperties", 
+                        "DPN"
+                    }, 
+
+                    Package (0x02)
+                    {
+                        "mipi-sdw-dp-4-source-subproperties", 
+                        "DPN"
+                    }, 
+
+                    Package (0x02)
+                    {
+                        "mipi-sdw-dp-5-sink-subproperties", 
+                        "DPN"
+                    }, 
+
+                    Package (0x02)
+                    {
+                        "mipi-sdw-dp-6-source-subproperties", 
+                        "DPN"
+                    }, 
+
+                    Package (0x02)
+                    {
+                        "mipi-sdw-dp-7-sink-subproperties", 
+                        "DPN"
+                    }, 

```

Comment:
that's a lot of ports, are they all valid?

Reply:
@plbossart I am not sure about it, will check with Realtek.